crates/rust-analyzer/src/flycheck.rs

```diff
@@ -232,13 +367,62 @@ enum FlycheckStatus {
     Finished,
 }
 
-pub(crate) const SAVED_FILE_PLACEHOLDER: &str = "$saved_file";
+/// This is stable behaviour. Don't change.
```

Comment:
fwiw, we can remove `$saved_file` with a bit of notice.

Reply:
actually, no, I lied we need to keep this for while. the main user of this is Fuchsia and until they're using Bazel, I don't think it's feasible to remove `$saved_file`.

Reply:
(cc: @P1n3appl3)

Reply:
thanks for the ping. We'd actually be ok with removing `$saved_file` if we could use `{label}` in the flycheck override. Today we actually do that mapping ourselves (wrapper script checks a global file list to find all labels that include the saved file and runs a check build/clippy on all those in the build system).

I'd be happy to migrate us over to label replacement when this lands.